Gajeongtongsin/Gajeongtongsin/Screens/Parent/Sending/SendingViewController.swift

```diff
@@ -137,20 +159,50 @@ class SendingViewController: BaseViewController {
         messageTypeButton.menu = UIMenu(options: .displayInline, children: [
             UIAction(title: "결석", handler: { _ in
                 self.messageTypeButton.setTitle("결석", for: .normal)
-//                self.textLabelDate.isHidden = false
-//                self.datePicker.isHidden = false
+                self.textLabelDate.isHidden = false
+                self.datePicker.isHidden = false
                 self.datePicker.datePickerMode = .date
-                print("결석 누름")
+                self.textLabelReason.isHidden = false
+                self.textFieldForReason.isHidden = false
+                self.sendButton.isHidden = false
             }),
             UIAction(title: "조퇴", handler: { _ in
                 self.messageTypeButton.setTitle("조퇴", for: .normal)
-//                self.textLabelDate.isHidden = false
-//                self.datePicker.isHidden = false
+                self.textLabelDate.isHidden = false
+                self.datePicker.isHidden = false
                 self.datePicker.datePickerMode = .dateAndTime
-                print("조퇴 누름")
+                self.textLabelReason.isHidden = false
+                self.textFieldForReason.isHidden = false
+                self.sendButton.isHidden = false
             })
         ])
         
+        //프로퍼티 옵저버 구현 고려 중 (날짜, 시간 값이 세팅되어야 사유 입력창 등장)
+//        var selectedDate: Date = datePicker.date {
+//            didSet {
+//                self.textLabelReason.isHidden = false
+//                self.textFieldForReason.isHidden = false
+//            }
+//        }
+    }
+    
+    func msgType() -> MessageType {
+        let msgType = messageTypeButton.currentTitle == "결석" ? MessageType.absence : MessageType.earlyLeave
+        return msgType
+    }
+  
+    @objc func sendMessage() {
+        let newMsg = Message(type: msgType(),
+                             sentDate: Date(),
+                             expectedDate: "\(datePicker.date)",
+                             content: textFieldForReason.text ?? "",
+                             isCompleted: false)
+        currentParent.sendingMessages.append(newMsg)
```

Comment:
```suggestion
        mainTeacher.parentUserIds[0].parensendingMessages.append(newMsg)
```
로 바꿔주시면 데이터가  Mock데이터에 추가됩니다!

Reply:
수정했습니다!